views/admin/javascripts/exhibits.js

```diff
@@ -173,26 +191,15 @@ Omeka.ExhibitBuilder = {};
             });
         }
 
-        function setSearchVisibility(show) {
-            var searchForm = $('#page-search-form');
-            var searchButton = $('#show-or-hide-search');
-
-            if (typeof show === 'undefined') {
-                show = !searchForm.is(':visible');
-            }
-            if (show) {
-                searchForm.show();
-                searchButton.addClass('hide-form').removeClass('show-form');
-            } else {
-                searchForm.hide();
-                searchButton.addClass('show-form').removeClass('hide-form');
-            }
-        }
 
         /**
          * Use AJAX to load the form for an attachment.
          */
-        this.loadItemOptionsForm = function(data) {
+        this.loadItemOptionsForm = function(data, ajaxUrl) {
```

Comment:
No, sorry, that was some leftover cruft from some refactoring that didn't work out.  It should be gone now.